### PR TITLE
created a new class clalled AtlasResponse

### PR DESCRIPTION
**Pull request by ljeyaratnam**
- which can be used to return id when posting to atlas

### CODE REVIEW

File: src/main/java/org/atlasapi/media/entity/simple/response/AtlasResponse.java [hunk 1]
```diff
@@ -0,0 +1,15 @@
+package org.atlasapi.media.entity.simple.response;
+
+
+public class AtlasResponse {
+
+    private final String id;
```
Comment (Spartakovic) [inline src/main/java/org/atlasapi/media/entity/simple/response/AtlasResponse.java]: maybe we need to add a new field for location?

**Comment by MartinsIrbe** [comment]
LGTM

File: src/main/java/org/atlasapi/media/entity/simple/response/AtlasResponse.java [hunk 2]
```diff
@@ -0,0 +1,15 @@
+package org.atlasapi.media.entity.simple.response;
+
+
+public class AtlasResponse {
```
Comment (Dalamar42) [inline src/main/java/org/atlasapi/media/entity/simple/response/AtlasResponse.java]: Maybe `UpdateResponse` might be a better name? `AtlasResponse` is a bit too generic for what this does

Comment (Dalamar42) [inline src/main/java/org/atlasapi/media/entity/simple/response/AtlasResponse.java]: Or `WriteResponse` actually since it could be update or create

**Pull request merged**